content/influxdb/v2.1/upgrade/v2-to-v2.md

```diff
@@ -0,0 +1,82 @@
+---
```

Comment:
Most of the content from the old `v2-beta-to-v2.md` seems to have been lost here? Upgrading from <= 2.0-beta16 still requires the manual steps. Upgrading from 2.0-GA+ should Just Work by dropping a new binary into place

Reply:
@danxmoran In the steps listed below, I note that to upgrade from 2.0-beta-16 or older, they should first upgrade to InfluxDB 2.0 (I provide a link to the manual upgrade instructions). I did this for the following reasons:

- The number of users who will be updating from a 2.0-beta version to 2.1 will probably be VERY small.
- Not having the manual upgrade steps required for 2.0-beta-16  or earlier makes 2.1 and all future releases a little easier to maintain.